### PR TITLE
added clear 3scale option explanation

### PR DESCRIPTION
**Pull request by neal-timpe**
https://bugzilla.redhat.com/show_bug.cgi?id=1753318 @yxun can you take a look and make sure this is enough?

### CODE REVIEW

**Comment by yxun** [comment]
> https://bugzilla.redhat.com/show_bug.cgi?id=1753318 @yxun can you take a look and make sure this is enough?

Hi Neal, I think this is enough.  I don't have detail information about 3scale. @brian-avery @knrc do you think any other information Neal need to add related to the 3scale option ?

**Comment by brian-avery** [comment]
Looks good to me, but I'm not sure if any other content is needed.

**Comment by neal-timpe** [comment]
@openshift/team-documentation Can you take a look? This is going to 4.1, 4.2, and beyond.

File: service_mesh/threescale_adapter/threescale-adapter.adoc [hunk 1]
```diff
@@ -4,7 +4,8 @@ include::modules/ossm-document-attributes.adoc[]
 :context: threescale-adapter
 toc::[]
 
-The 3scale Istio Adapter allows you to label a service running within the {ProductName} and integrate that service with the 3scale API Management solution.
+The 3scale Istio Adapter is an optional adapter that allows you to label a service running within the {ProductName} and integrate that service with the 3scale API Management solution.
+It's not required for {ProductName}.
```
Comment (bergerhoffer) [inline service_mesh/threescale_adapter/threescale-adapter.adoc]: It's -> It is

**Review by bergerhoffer — COMMENTED** [review]
Just one small thing, otherwise LGTM!

**Pull request merged**